style.css

```diff
@@ -106,6 +106,8 @@ p {
 }
 
 .skill {
+  display: grid;
+  grid-template-columns: auto;
   align-items: center;
```

Comment:
The same applies for this section display it to flex instead of grid. We only display grid to a main container of a page not to its children elements.